types/office-js-preview/index.d.ts

```diff
@@ -18491,14 +18547,266 @@ declare namespace Office {
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
          *
-         * @param sensitivity - The sensitivity value as enum or string.
+         * @param sensitivity - The sensitivity level as an enum or string.
          * @param callback - Optional. When the method completes, the function passed in the `callback` parameter is called with a single parameter,
          *                `asyncResult`, which is an `Office.AsyncResult` object.
          *
          * @beta
          */
         setAsync(sensitivity: MailboxEnums.AppointmentSensitivityType | string, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
     }
+    /**
+     * Provides methods to get or set a message or appointment's {@link https://learn.microsoft.com/microsoft-365/compliance/sensitivity-labels | sensitivity label}.
+     * 
+     * @remarks
+     * [Api set: Mailbox preview]
+     * 
+     * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+     *
+     * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+     * 
+     * @beta
+     */
+    interface SensitivityLabel {
+        /**
+         * Gets the unique identifier (GUID) of the sensitivity label applied to a message or appointment being composed.
+         * 
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * @param options - An object literal that contains one or more of the following properties:-
```

Comment:
What's up with the ":-" here and elsewhere?

Reply:
I believe the use of ":-" was decided on to provide more information on the `options` properties due to some formatting limitations in DT. I'll follow up on this, as I think this may change in the future.